src/EnergyPlus/RuntimeLanguageProcessor.cc

```diff
@@ -1633,7 +1694,10 @@ namespace RuntimeLanguageProcessor {
 				if ( DeveloperFlag ) gio::write( OutputFileDebug, fmtA ) << "PAREN \"" + StringToken + "\"";
 				Token( NumTokens ).Type = TokenParenthesis;
 				Token( NumTokens ).String = StringToken;
-				if ( NextChar == '(' ) Token( NumTokens ).Parenthesis = ParenthesisLeft;
+				if ( NextChar == '(' ) {
+					Token( NumTokens ).Parenthesis = ParenthesisLeft;
+					OperatorProcessing = true;
+				}
```

Comment:
use same logic for ( as used for *, i.e., this character requires that OperatorProcessing be true in case there is a trailing minus